.github/workflows/ci.yml

```
@@ -48,3 +48,14 @@ jobs:
             unlist()
           warnings_found <- any(grepl("expectation_warning", expectations))
           quit(status = warnings_found)
+      - name: Spell Check
```

Blank line will help readability here.
```suggestion

      - name: Spellcheck
```